### PR TITLE
Add user to organization when creating it.

### PR DESCRIPTION
**Pull request by pmeijer**
Adding orgId is a parameter when creating org.

### CODE REVIEW

File: src/server/api/index.js [hunk 1]
```diff
@@ -391,7 +391,7 @@ function createAPI(app, mountPath, middlewareOpts) {
             });
     });
 
-    router.put('/orgs', function (req, res, next) {
+    router.put('/orgs:orgId', function (req, res, next) {
```
Comment (lattmann) [inline src/server/api/index.js]: Should not this be `'/orgs/:orgId'`?
Are not we missing a `/`?

File: test/server/api/index.spec.js [hunk 2]
```diff
@@ -998,26 +998,27 @@ describe('API', function () {
...
                     }
                 };
 
-                agent.get(server.getUrl() + '/api/v1/orgs/' + newOrg.orgId)
+                agent.get(server.getUrl() + '/api/v1/orgs/' + orgId)
                     .end(function (err, res) {
                         expect(res.status).equal(404, err); // user should not exist at this point
 
-                        agent.put(server.getUrl() + '/api/v1/orgs')
+                        agent.put(server.getUrl() + '/api/v1/orgs' + orgId)
```
Comment (lattmann) [inline test/server/api/index.spec.js]: Should not this be `'/api/v1/orgs/' + orgId`?

File: test/server/api/index.spec.js [hunk 3]
```diff
@@ -1026,27 +1027,28 @@ describe('API', function () {
...
                         }
                     };
 
-                    agent.get(server.getUrl() + '/api/v1/orgs/' + newOrg.orgId)
+                    agent.get(server.getUrl() + '/api/v1/orgs/' + orgId)
                         .end(function (err, res) {
                             expect(res.status).equal(404, err); // org should not exist at this point
 
-                            agent.put(server.getUrl() + '/api/v1/orgs')
+                            agent.put(server.getUrl() + '/api/v1/orgs' + orgId)
```
Comment (lattmann) [inline test/server/api/index.spec.js]: Should not this be `'/api/v1/orgs/' + orgId`?

File: test/server/api/index.spec.js [hunk 4]
```diff
@@ -1056,18 +1058,18 @@ describe('API', function () {
...
```
Comment (lattmann) [inline test/server/api/index.spec.js]: Should not this be `'/api/v1/orgs/' + orgId`?

File: test/server/api/index.spec.js [hunk 5]
```diff
@@ -1081,18 +1083,18 @@ describe('API', function () {
...
                         }
                     };
 
-                    agent.get(server.getUrl() + '/api/v1/orgs/' + newOrg.orgId)
+                    agent.get(server.getUrl() + '/api/v1/orgs/' + orgId)
                         .end(function (err, res) {
                             expect(res.status).equal(200, err);
 
-                            agent.put(server.getUrl() + '/api/v1/orgs')
+                            agent.put(server.getUrl() + '/api/v1/orgs' + orgId)
```
Comment (lattmann) [inline test/server/api/index.spec.js]: Should not this be `'/api/v1/orgs/' + orgId`?

**Comment by pmeijer** [comment]
Good catch, thanks! It's fixed now.

**Pull request merged**